package.json

```diff
@@ -4,11 +4,10 @@
   "description": "JavaScript development environment Pluralsight course by Cory House",
   "scripts": {
     "prestart": "babel-node buildScripts/startMessage.js",
-    "start": "npm-run-all --parallel security-check open:src lint:watch test:watch start-mockapi",
+    "start": "npm-run-all --parallel open:src lint:watch test:watch start-mockapi",
```

Comment:
Node security project (NSP) is now built into npm, so you don't need to reference it or call it manually anymore. 😀